pkg/controllers/provisioning/scheduling/scheduler.go

```diff
@@ -205,10 +209,19 @@ func (s *Scheduler) Solve(ctx context.Context, pods []*corev1.Pod) Results {
 	errors := map[*corev1.Pod]error{}
 	QueueDepth.DeletePartialMatch(prometheus.Labels{ControllerLabel: injection.GetControllerName(ctx)}) // Reset the metric for the controller, so we don't keep old ids around
 	q := NewQueue(pods...)
+
+	startTime := s.clock.Now()
+	lastLogTime := s.clock.Now()
+	batchSize := len(q.pods)
 	for {
 		QueueDepth.With(
 			prometheus.Labels{ControllerLabel: injection.GetControllerName(ctx), schedulingIDLabel: string(s.id)},
 		).Set(float64(len(q.pods)))
+
+		if s.clock.Since(lastLogTime) > time.Minute {
+			log.FromContext(ctx).Info("printing scheduling progress", "pods-scheduled", batchSize-len(q.pods), "pods-left", len(q.pods), "duration", s.clock.Since(startTime).Truncate(time.Second))
```

Comment:
Should this be a Debug log? If we were only logging this for the provisioner, I think it's rare enough that keeping it as Info is appropriate, but I'm wondering if it will be spammy due to parallel disruption.

Reply:
2 a minute at peak times I don't think is spammy

Reply:
We discussed -- we don't add a real logger for any of our disruption loops so this isn't going to print anything out for anything but provisioning.